src/analytics_url.coffee

```diff
@@ -0,0 +1,42 @@
+define [
+  'helpers/url_helper'
+  'helpers/base64_helper'
+], (URLHelper, Base64)->
+  class AnalyticsUrl
+    URL_PARAM_NAME = 'skr_info'
+    TTL = 60 * 1000 # milliseconds
+
+    constructor: (@url) ->
+
+    format_params: (mode, session, metadata = {}) ->
+      params = [session, @_expiration(), metadata]
```

Comment:
Maybe we should use a js object here instead of Array. 🤔

What do you think?

Reply:
I had it as a JS object but it was almost double the length..